comp/process/runner/component.go

```diff
@@ -29,3 +32,11 @@ type Component interface {
 var Module = fxutil.Component(
 	fx.Provide(newRunner),
 )
+
+// DisableContainerFeaturesForTest is an fx option that disables all container features.
+// For some platforms, container features don't work, so in order to keep the tests from crashing we must disable container
+// features.
+var DisableContainerFeaturesForTest = fx.Invoke(func(t testing.TB, _ configComp.Component) {
```

Comment:
So I was originally trying to reuse this, but couldn't find a great way to do it. I ended up moving this to `comp/process/utils/test_utils.go`. I however, did not add the `test` build tag to it, since then the only way to test would be to use the invoke command, which I didn't want to make a requirement. I think this solution should be cleaner, and this way we can reuse the option.